tracing-example/pom.xml

```diff
@@ -34,10 +34,10 @@
     <properties>
         <version.jakarta.ee>10.0.0</version.jakarta.ee>
         <version.org.jboss.arquillian>1.7.0.Alpha14</version.org.jboss.arquillian>
-        <version.org.jboss.resteasy>6.2.2.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy>6.2.3.Final</version.org.jboss.resteasy>
```

Comment:
This looks strange to me. Upstream already has this upgrade https://github.com/resteasy/resteasy-examples/blob/main/tracing-example/pom.xml#L37. Maybe you just need a rebase? Still strange how git shows this though.

Reply:
Yeah it's really weird :D I have rebased+force push the branch.